admin/common_auth/views.py

```diff
@@ -47,3 +55,45 @@ def get_success_url(self):
 def logout_user(request):
     logout(request)
     return redirect('auth:login')
+
+
+class RegisterUser(SuperUser, FormView):
+    form_class = UserRegistrationForm
+    template_name = 'register.html'
+
+    def form_valid(self, form):
+        osf_id = form.cleaned_data.get('osf_id')
+        try:
+            osf_user = User.load(osf_id)
+            osf_user.system_tags.append(PREREG_ADMIN_TAG)
+        except AttributeError:
+            return page_not_found(
```

Comment:
Again, I'd raise `Http404('OSF user with id "{}" not found. Please double check.'.format(osf_id))` just for modularity.